### PR TITLE
[Gubernator] Show more PR context

### PR DESCRIPTION
**Pull request by rmmh**
This groups builds for a PR into columns based on the versions being tested.

Also, timestamps are now localized to US/Pacific by default, or whatever your browser's local time is.

### CODE REVIEW

File: gubernator/pull_request.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        headings: a list of [(version, column width, timestamp)]
+        rows: a list of [(build, [(number, status) or None])]
+    """
+
+    def commit(version):
+         return version.split('+')[-1]
+
+    # Compute the headings first -- versions and their maximum build counts.
+
+    versions = {}
```
Comment (spxtr) [inline gubernator/pull_request.py]: In lieu of types, I would appreciate a more detailed comment saying what these two dicts are.

File: gubernator/static/build.js [hunk 2]
```diff
@@ -11,3 +11,20 @@ function select(node) {
 	sel.removeAllRanges();
 	sel.addRange(range);
 }
+
+// Rewrite timestamps to respect the current locale.
```
Comment (spxtr) [inline gubernator/static/build.js]: This feels like a bit of a hack but I can't think of a better way.

**Comment by spxtr** [comment]
LGTM, as far as I can tell.

File: gubernator/pull_request.py (hunk 1 above)
Comment (rmmh) [inline gubernator/pull_request.py]: done.

**Pull request merged**